paddle/operators/roi_pool_op.cu

```diff
@@ -0,0 +1,265 @@
+/* Copyright (c) 2016 PaddlePaddle Authors. All Rights Reserve.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. */
+
+#include "paddle/platform/cuda_helper.h"
+#include "paddle/operators/roi_pool_op.h"
+
+namespace paddle {
+namespace operators {
+
+#define FLT_MAX __FLT_MAX__
```

Comment:
The use of `__FLT_MAX__` is not a good idea because it is a [GCC-specific predefined macro](https://gcc.gnu.org/onlinedocs/cpp/Predefined-Macros.html#Predefined-Macros).  If we are using another compiler other than GCC, it is probable that we don't have `__FLT_MAX__` defined.

It seems to me that the solution could be using standard C library

```c
#include <float.h>
printf("%f", FLT_MAX);
```

or the standard C++ library

```c++
#include <limits>
printf("%f", numeric_limits<float>::max());
```

This problem doesn't block the merge of this PR. I created an [issue](https://github.com/PaddlePaddle/Paddle/issues/5850) reminding us to fix it.